build.sbt

```diff
@@ -7,6 +7,6 @@ crossScalaVersions := Seq(scalaVersion.value, "2.12.10")
 releaseCrossBuild := true
 
 libraryDependencies ++= Seq(
-  "is.cir" %% "ciris" % "1.0.1",
+  "is.cir" %% "ciris" % "1.0.4+62-0d574645+20200422-1805-SNAPSHOT",
```

Comment:
Isn't 1.0.4 sufficient for those changes?

Reply:
No, as per the note in the pull request:
> Note this still relies on an unpublished snapshot version of Ciris with support for `Resource`.

we will have to wait for at least 1.0.5 (assuming the changes can be made binary compatible). I'll update the pull request once we have the changes merged in Ciris (still no pull request available).